doc/api/n-api.md

```diff
@@ -4755,6 +4755,9 @@ is sufficient and appropriate. Use of the `napi_make_callback` function
 may be required when implementing custom async behavior that does not use
 `napi_create_async_work`.
 
+Also note that any `process.nextTick`s scheduled by JavaScript during the callback
+are ran before returning back to C/C++.
```

Comment:
Can you also mention promises?

Reply:
Is the microtask queue ran before returning to c/c++ as well?